master/cli-reference/ark_describe_restores.md~ba18ad8... add v0.9.0 docs to site

```diff
@@ -0,0 +1,39 @@
+## ark describe restores
```

Comment:
This looks like a garbage file (git leftover from something or other), should be removed